### PR TITLE
Fix IDE0090: Simplify new expression part 7

### PR DESCRIPTION
**Pull request by xtqqczze**
https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0090

### CODE REVIEW

**Review by iSazonov — COMMENTED** [review]
@xtqqczze I stopped review - too many unrelated changes.

File: src/Microsoft.PowerShell.Security/security/AclCommands.cs [hunk 1]
```diff
@@ -385,8 +362,7 @@ public static SecurityIdentifier GetCentralAccessPolicyId(PSObject instance)
                     return null;
                 }
 
-                NativeMethods.ACL sacl = new NativeMethods.ACL();
```
[the remark this answers is not in the document]
Comment (xtqqczze) [reply src/Microsoft.PowerShell.Security/security/AclCommands.cs:388]: Opened #14358 for these changes,

[the remark this answers is not in the document]
Comment (iSazonov) [reply src/Microsoft.PowerShell.Security/security/AclCommands.cs:388]: Can we continue?

[the remark this answers is not in the document]
Comment (xtqqczze) [reply src/Microsoft.PowerShell.Security/security/AclCommands.cs]: I've reverted unrelated changes

**Review by iSazonov — APPROVED** [review]

**Pull request merged**